src/services/ui/src/pages/dashboard/index.tsx

```diff
@@ -50,7 +50,7 @@ export const Dashboard = () => {
             value={query.state.tab}
             onValueChange={(tab) =>
               query.onSet(
-                (s) => ({ ...s, tab: tab as OsTab, search: "" }),
+                (s) => ({ ...s, filters: [], tab: tab as OsTab, search: "" }),
```

Comment:
Need to reset the filters on tab change, since filter options will differ between spas - waivers